gradle/dependencies.gradle

```diff
@@ -116,7 +116,7 @@ versions += [
   testRetryPlugin: "1.1.6",
   zinc: "1.3.5",
   zookeeper: "3.5.8",
-  zstd: "1.4.5-6"
+  zstd: "1.4.5-8"
```

Comment:
This change means we are using the recycling buffer pool, right? I think we should be passing our own buffer pool.

Reply:
Ah, you explained this in the PR description. Let me think about that.